scripts/test-ts-definition-files.ts

```diff
@@ -83,48 +83,57 @@ function packPuppeteer() {
 const tar = packPuppeteer();
 const tarPath = path.join(process.cwd(), tar);
 
+function cdIntoProjectAndRunCommand(projectLocation: string, command: string) {
+  return spawnSync(
+    'sh',
+    ['-c', [`cd "${projectLocation}"`, command].join('; ')],
```

Comment:
We're essentially moving the problem to this one "command" string that we pass to `sh -c` now. But we're still hardcoding the arguments within that command string ourselves, so this still doesn't escape correctly.

Taking a step back, I don't think we need this custom logic at all. Have you considered using `spawnSync(command, { cwd: projectLocation })` instead of creating this helper?